spark_log_parser/parsing_models/application_model.py

```diff
@@ -329,15 +295,38 @@ def __init__(self, eventlogpath, bucket=None, stdoutpath=None, debug=False):  #
         self.num_instances = len(numpy.unique(hosts))
```

Comment:
This can just be `len(hosts)` since you changed it to a set.

Reply:
ah, right you are :)